csss-site/src/events/tech_fair/templates/tech_fair/2022/tech_fair_main.html

```diff
@@ -0,0 +1,186 @@
+{% load staticfiles %}
```

Comment:
fix https://validator.w3.org/nu/?doc=https%3A%2F%2Fdev.sfucsss.org%2FPR-470%2Fevents%2Ftech_fair%2F2022%2Fmain

Reply:
Duplicated IDs addressed in newest commit.